src/legacy/core_plugins/elasticsearch/index.js

```diff
@@ -19,14 +19,12 @@
 import { first } from 'rxjs/operators';
 import { Cluster } from './server/lib/cluster';
 import { createProxy } from './server/lib/create_proxy';
-import { handleESError } from './server/lib/handle_es_error';
-import { versionHealthCheck } from './lib/version_health_check';
 
 export default function (kibana) {
   let defaultVars;
 
   return new kibana.Plugin({
-    require: ['kibana'],
+    require: [],
```

Comment:
Wouldn't it affect an execution order?

Reply:
Yea, the `kibana`  legacy plugin is no longer assured to be initialized before the `es` one. However as the kibana plugin is an empty shell now, this shouldn't have any consequences (and is preparing the actual removal of the `kibana` plugin)